scripts/start.sh

```diff
@@ -47,7 +47,14 @@ echo Checking whether GAE is installed in $GOOGLE_APP_ENGINE_HOME
 if [ ! -f "$GOOGLE_APP_ENGINE_HOME/appcfg.py" ]; then
   echo "Installing Google App Engine (this may take a little while)..."
   mkdir -p $GOOGLE_APP_ENGINE_HOME
+  echo "Downloading..."
   curl --silent https://storage.googleapis.com/appengine-sdks/deprecated/1919/google_appengine_1.9.19.zip -o gae-download.zip
+  if [ 0 -eq $? ];
+  then
+	  echo "Downloaded. Unzipping..."
+  else
+	  echo "Error in downloading Google App Engine"
```

Comment:
Ah, I see -- thanks for clarifying.

I suggest not doing that in this PR, but if you like, please feel free to file a separate issue documenting additional places where additional error messages are required, and we can treat that separately. It is also totally fine to not do that -- this hasn't been much of a problem historically, and we can look into issues as they come up.